request-server.go

```diff
@@ -161,7 +161,9 @@ func (rs *RequestServer) packetWorker(
 			handle := pkt.getHandle()
 			rpkt = statusFromError(pkt, rs.closeRequest(handle))
 		case *sshFxpRealpathPacket:
```

Comment:
This case is not correct and ultimately not needed. The sshFxpRealpathPacket was special cased as it was only cleaning the path and returning it, without calling the request handler code. Your change makes it so that it wants to be passed to the request handler code, to do that you'd just delete this case. As is it returns the sshFxpHandlePacket instead of the sshFxpNamePacket, which is wrong. Anyways, just deleting this case fixes that.

Reply:
You are right. I will review it. Thanks.